### PR TITLE
Updated contributing tab in Readme!

### PR DESCRIPTION
**Pull request by ssiddhantsharma**
### Description
Created a modern contributing tab with badges.

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] Community content

### Checklist:

- [ ] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 [![Docs](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)
 [![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
-
```
Comment (shahidhk) [inline README.md]: Please add these two lines back.

File: README.md [hunk 2]
```diff
@@ -154,9 +152,10 @@ We are committed to fostering an open and welcoming environment in the community
 
 If you want to report a security issue, please [read this](SECURITY.md).
 
-## Contributing
+## Contributing 
+We :heart: contributions from everyone! We welcome [bug reports](https://github.com/hasura/graphql-engine/issues/), [bug fixes](https://github.com/hasura/graphql-engine/pulls) and feedback on the library is always appreciated. Look at the [Contributor Guidelines](CONTRIBUTING.md) for more details.
```
Comment (shahidhk) [inline README.md]:
```suggestion
We :heart: contributions from everyone! Check out our [contributing guide](CONTRIBUTING.md) for more details.
```

File: README.md [hunk 3]
```diff
@@ -154,9 +152,10 @@ We are committed to fostering an open and welcoming environment in the community
...
+## [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/hasura/graphql-engine/issues) [![GitHub contributors](https://img.shields.io/github/contributors/hasura/graphql-engine.svg)](https://GitHub.com/hasura/graphql-engine/graphs/contributors/)
```
Comment (shahidhk) [inline README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (ssiddhantsharma) [reply README.md]: Resolved the issue!

File: README.md (hunk 2 above)
Comment (ssiddhantsharma) [reply README.md]: Resolved the issue!

**Comment by shahidhk** [comment]
@ssiddhantsharma Checkout the diff, a sentence is repeated twice.

**Comment by shahidhk** [comment]
Hi @ssiddhantsharma, thanks for the contribution. Let's re-open the PR when you update the required changes. :smile:

**Pull request closed without merging**